cypress/e2e/customers.cy.js

```diff
@@ -0,0 +1,142 @@
+
+describe('GET /customers', () => {
+
+  beforeEach(() => {
+    //  cy.visit('/');
+  })
```

Comment:
```suggestion
  })

```

As mentioned just before, here's a good place for an empty line.